### PR TITLE
:bar_chart: igh: Upload homelessness data from the Institute of Global Homelessness

### PR DESCRIPTION
**Pull request by paarriagadap**
Main issue https://github.com/owid/owid-issues/issues/791

- Within the IGH data we will focus in homeless definitions mostly and not the data itself, given that any cross-country comparison is straightforward.
- I have modified the cookie cutter file in Wizard-garden, to add recently included metadata parameters. I also sorted them according to more typical use (`variable.display`).
- Additionally, I created a separate chartbook dag file to not mix it with poverty_inequality.

### CODE REVIEW

File: etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        presentation:
+          title_public: Number of people experiencing homelessness
+        display:
+          name: Number of people experiencing homelessness
+          numDecimalPlaces: 0
+          <<: *common_display
+
+      people_homeless_per_10k:
+        title: People experiencing homelessness per 10,000 people
+        unit: people
```
Comment (spoonerf) [inline etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml:36]: Nit pick but perhaps the unit here should acknowledge that it's a rate? Although maybe hard to do without having a really long unit...

File: etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+        type: ordinal
+        sort:
+          - "Yes"
+          - "No"
+
+      total_homeless_data_scorecard:
+        title: Homeless Data Scorecard
+        unit: ""
+        short_unit: ""
+        description_short: The Homeless Data Scorecard is an 11-point scale that provides an assessment of the current state of a country’s enumeration and data practices on homelessness.
```
Comment (spoonerf) [inline etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml:297]: Might be worth just noting whether 11 = best or worst.

**Review by spoonerf — APPROVED** [review]
Looks good!

File: etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+      people_experiencing_homelessness:
+        title: Number of people experiencing homelessness
+        unit: people
+        short_unit: ""
+        description_short: Due to different definitions and data collection methods, the estimated number of people experiencing homelessness is not directly comparable across countries.
+        presentation:
+          title_public: Number of people experiencing homelessness
+        display:
+          name: Number of people experiencing homelessness
+          numDecimalPlaces: 0
```
[the remark this answers is not in the document]
Comment (paarriagadap) [reply etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml:31]: It's integer. Maybe you copied from the other one

File: etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml (hunk 1 above)
Comment (paarriagadap) [reply etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml:36]: I will change it for `people in homelessness per 100,000 population`. Thanks!

File: etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (spoonerf) [reply etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml:31]: Whoops yep my bad!

File: snapshots/igh/2024-07-05/better_data_homelessness.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
+"""
+The file has been provided by the IGH team by email on 2024-07-04. It is not available online.
```
[the remark this answers is not in the document]
Comment (paarriagadap) [reply snapshots/igh/2024-07-05/better_data_homelessness.py:2]: That's a good idea, thanks!

**Comment by paarriagadap** [comment]
Thank you, @spoonerf!

**Pull request merged**